lib/desktop/menu/parser.ex

```diff
@@ -150,7 +150,9 @@ defmodule Desktop.Menu.Parser do
     Enum.filter(content, &(not is_binary(&1)))
   end
 
-  defp error(message, meta \\ %{line: 0, column: 0}) do
+  @spec error(binary()) :: no_return()
```

Comment:
Instead of `binary` I believe you mean `String.t()` for this specific reason. We are not expecting binary (code) input here, but instead something text, right?